src/common/components/map/controls/DrawControl.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useContext } from 'react';
+import { IconPlusCircle, IconLinkExternal } from 'hds-react';
+import clsx from 'clsx';
+import { $enum } from 'ts-enum-util';
+import ControlPanel from './ControlPanel';
+import styles from './Controls.module.scss';
+import MapContext from '../MapContext';
+import { DrawTool } from '../constants';
+
+const getDrawIcon = (drawTool: DrawTool) => {
+  switch (drawTool) {
+    case DrawTool.CIRCLE:
+      return <IconPlusCircle size="m" aria-hidden="true" />;
```

Comment:
These are temporary icons. We should implement somekind of custom icon system because HDS doesnt include correct icons.